### ISSUE TITLE
Customization Suggestion

### ISSUE THREAD

**Issue by WesleyjanLacerda** [dchapyshev/aspia#285]
Firstly, I need to point out that this application is incredible, something out of this world, really good, I would like to leave some suggestions if possible, in this case:

- Customize the title of the host application for example, to make it a little more friendly for the end user, it could be a json file, next to the .exe for example, with information about the application title for example, or icons too if possible

![image](https://github.com/dchapyshev/aspia/assets/74305459/17581a8d-d8ed-44b7-a896-8ef0233915c0)

- Another point would be if it were possible, the password generator would be shorter than 6 digits, I imagine that 6 is the minimum for security reasons, but in the case of small companies with many end users, a shorter password would speed up the use of the application

- Another question too, but now in the "Aspia Client" there would be both the possibility of changing the application title or icons...

- And also a possibility to leave the "pre-saved, or pre-filled, or pre-defined", User and password" or just the User, I say more because in the profile to use practically the same user and password in several instances , the connection with the end user would become more valuable

![image](https://github.com/dchapyshev/aspia/assets/74305459/2dd68b6c-c974-4f96-9b97-4b79a43fd2dd)

Again, I really appreciate this incredible platform, really great show!

**Comment by felipeborela** [comment]
You can program the same password on all your Hosts (Aspia Host)

In Aspia Console, open the "address book properties", and there you can set the default password in the "Default Configuration" tab

During Host registration in Aspia Console, you can check the "Inherit from parent" box

![image](https://github.com/dchapyshev/aspia/assets/9823628/7e3701be-59dd-47c4-b06a-6022b1a004de)
![image](https://github.com/dchapyshev/aspia/assets/9823628/05c4d430-ebfd-4f53-9946-d80616e94bae)

**Comment by WesleyjanLacerda** [comment]
> Você pode programar a mesma senha em todos os seus Hosts (Aspia Host)
> 
> No Aspia Console, abra as "propriedades do catálogo de endereços", e lá você pode definir a senha padrão na aba "Configuração Padrão"
> 
> Durante o registro do Host no Aspia Console, você pode marcar a caixa "Herdar do pai"
> 
> ![imagem](https://private-user-images.githubusercontent.com/9823628/311786507-7e3701be-59dd-47c4-b06a-6022b1a004de.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTAxNzYzODcsIm5iZiI6MTcxMDE3NjA4NywicGF0aCI6Ii85ODIzNjI4LzMxMTc4NjUwNy03ZTM3MDFiZS01OWRkLTQ3YzQtYjA2YS02MDIyYjFhMDA0ZGUucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDMxMSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAzMTFUMTY1NDQ3WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZjU4MTg1ZTBhNTM4MTIxYjNlOWE1YjhhNjg0MTUyOGRiZWJkMjUxOGNjOThmNDg3MDBkYTE1OGUyYTAwN2FiNCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.G3rcqJNHEZEaNuN_SPLXHb4J7lSID_s4aHL_Nd9bovs) ![imagem](https://private-user-images.githubusercontent.com/9823628/311786405-05c4d430-ebfd-4f53-9946-d80616e94bae.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTAxNzYzODcsIm5iZiI6MTcxMDE3NjA4NywicGF0aCI6Ii85ODIzNjI4LzMxMTc4NjQwNS0wNWM0ZDQzMC1lYmZkLTRmNTMtOTk0Ni1kODA2MTZlOTRiYWUucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDMxMSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAzMTFUMTY1NDQ3WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NDg1ODFjNTMxZDkzZGFmODJkZWFlMzhmNzBkYjFmNTQ0ZGQzMzFmZDIyZTRjN2U5NDM4NTU2ODNiMDgwNzAxZCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.w4Y9zv_CM7o5z4Qwcao6krVqcfy7cn0BdoWabZvSGvU)

Yes, I saw this option, but this is in the "Aspia Console", as we are several collaborators, it is bad to leave the "aspia_console" to everyone, so we are opting to use the "Aspia Client", because then each collaborator only accesses the other remote computer, without having to access the "Aspia Console" which has access to "all computers on the network"
